src/SqlPersistence/Saga/SqlDialect.cs

```diff
@@ -24,5 +24,7 @@ internal virtual object BuildSagaData(CommandWrapper command, RuntimeSagaInfo sa
         {
             return sagaInfo.ToJson(sagaData);
         }
+
+        internal abstract void ValidateJsonSettings(Newtonsoft.Json.JsonSerializer jsonSerializer);
```

Comment:
Why not make this virtual?

Reply:
Is there a reason the serializer is fully qualified here?

Reply:
clash with NServiceBus.JsonSerializer